server/models/User.coffee

```diff
@@ -463,7 +463,7 @@ UserSchema.statics.editableProperties = [
   'testGroupNumber', 'music', 'hourOfCode', 'hourOfCodeComplete', 'preferredLanguage',
   'wizard', 'aceConfig', 'autocastDelay', 'lastLevel', 'jobProfile', 'savedEmployerFilterAlerts',
   'heroConfig', 'iosIdentifierForVendor', 'siteref', 'referrer', 'schoolName', 'role', 'birthday',
-  'enrollmentRequestSent'
+  'enrollmentRequestSent', 'signupQueryParams'
```

Comment:
We probably will need to index by these, too. Not sure which ones yet, but let's take some action so we don't forget. (A TODO in the indexes section of `User.coffee`? Something else?)